src/otx/config/data/mmdet.yaml

```
@@ -0,0 +1,14 @@
+defaults:
+  - default
+
+data_format: coco_instances
```

```suggestion
datumaro_data_format: coco_instances
```
Since the filename is the `mmdet.yaml`, users could easily imagine the `data_format` will represents the data format of the `mmdet` instead of `Datumaro`. 

+ If we could add data_format examples, users could easily take the dataset format. (i.e., for my case, I spent some time finding the exact name (imagenet_with_subset_dirs) which has the same functionality `Torchvision.ImageFolder`.

---

That will be removed in the future. The reason why this exists is just I didn't have the time to implement before PoC. This seems not preventing you guys from doing P1 targets (migrating Geti support models). So, let's leave it as is.

Ultimately, I imagine that we will have the following data pipeline with Datumaro `detect` and `AnnotationType`.

1. Detect `data_format` in the given `data_root` (Datumaro).
2. Import the dataset from `data_root` with the detected `data_format`.
3. Get a set of annotation types existed in the imported dataset.
4. Decide whether the annotation type set obtained previously can do the given OTX Task.
     - For example, `OTXTaskType.DETECTION` can only train datasets having `Datum.AnnotationType.bbox`.